dependencies.toml

```diff
@@ -17,41 +17,41 @@ jetty93 = '9.3.30.v20211001'
 jetty94 = "9.4.48.v20220622"
 jmh = "1.35"
 json-unit = "2.35.0"
-junit5 = "5.8.2"
+junit5 = "5.9.0"
 kotlin = "1.7.10"
 kotlin-coroutine = "1.6.4"
-micrometer = "1.9.2"
+micrometer = "1.9.3"
 micrometer13 = "1.3.20"
-mockito = "4.6.1"
+mockito = "4.7.0"
 monix = "3.4.1"
 munit = "0.7.29"
 okhttp4 = "4.10.0"
 opensaml = "3.4.6"
 protobuf = "3.21.1"
 reactive-grpc = "1.2.3"
 reactive-streams = "1.0.4"
-reactor = "3.4.21"
+reactor = "3.4.22"
 resteasy = "5.0.4.Final"
 retrofit2 = "2.9.0"
-sangria = "3.0.1"
+sangria = "3.2.0"
 sangria-slowlog = "2.0.4"
 scala-java8-compat = "1.0.2"
 scalapb = "0.11.11"
 scalapb-json = "0.12.0"
 slf4j = "1.7.36"
 spring-boot1 = "1.5.22.RELEASE"
-spring-boot2 = "2.7.2"
+spring-boot2 = "2.7.3"
 tomcat8 = "8.5.81"
 tomcat9 = "9.0.65"
 
 [boms]
-brave = { module = "io.zipkin.brave:brave-bom", version = "5.13.10" }
-dropwizard-metrics = { module = "io.dropwizard.metrics:metrics-bom", version = "4.2.10" }
+brave = { module = "io.zipkin.brave:brave-bom", version = "5.13.11" }
```

Comment:
It was a hassle to find the location of versions back and forth in this file while upgrading versions.
I'd like to migrate embedded dependency versions into the `[versions]` table.
In addition to upgrading the versions, we can easily summarize the new dependencies in the PR description.